test/WebSites/InProcessForwardsCompatWebSite/InProcessWebSite.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\..\build\testsite.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.2</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\shared\**\*.cs" />
+    <Compile Include="..\InProcessWebSite\*.cs" />
```

Comment:
Use a recursive wildcard (`**\*.cs`)

Reply:
It will start pulling in obj output from the InProcessWebSite.